portfolio/src/main/webapp/functions/index.js

```diff
@@ -18,7 +33,13 @@ const createCommentElement = commentPayload => {
     const header = document.createElement("div");
     header.classList.add("header");
 
-    header.appendChild(username);
+    const tagsSection = document.createElement("div");
+    tagsSection.classList.add("header-tags");
+
+    tagsSection.appendChild(username);
+    tagsSection.appendChild(sentiment);
```

Comment:
I think it's pretty nifty that you're displaying your sentiment analysis usage. Though it might be interesting to have a sentiment summary page somewhere where you can see sentiment stats, but without the usernames/comments attached, instead of displaying it directly on the comment.

If user's can see their comments and the analysis, it might influence what comment they write before they even write it.

Not a change you need to do in this PR, though, and it's totally cool to leave it as is. The main purpose of the portfolio is to let you showcase your dev skills, so that's not lost here.

Reply:
Yeah, probably a separate dashboard with charts to visualize the sentiment of comments would be a great option to display the results of the comments.